Fabric.Identity.API/scripts/Register-Identity-IdPSS.ps1

```diff
@@ -13,6 +13,16 @@ param(
     [switch] $registerIdPSS
 )
 
+if(!($registerIdentity) -and !($registerIdPSS))
+{
+	Write-Host " You must register either Identity and/or IdPSS by adding their respective switch.
+	Examples:
+		.\Register-Identity-IdPSS.ps1 -registerIdentity
+		.\Register-Identity-IdPSS.ps1 -registerIdPSS
+		.\Register-Identity-IdPSS.ps1 -registerIdentity -registerIdPSS" -ForegroundColor Red -BackgroundColor Black
```

Comment:
Can just do a Write-DosMessage -Level "Fatal" to avoid having to set colors, it should also throw an error which should cause the script to exit

Reply:
Okay sounds good. I wasn't sure if I should throw an error or just exit. I changed it to `throw` which is basically shorthand for what you just described. Thanks for your input